src/js/utils/StringUtil.js

```diff
@@ -215,6 +215,29 @@ const StringUtil = {
     }, []);
 
     return this.idToTitle(splitID, splitBy, replace, removeConsecutive);
+  },
+
+  /**
+   * @param {String} str     - A string that we would like to trim.
+   * @param {String} trimStr - The string that we would like to trim from
+   *                         - the beginning and the end.
+   * @return {String}        - The trimmed string. If a non-string type is
+   *                         - entered it will just return the input.
+   */
+  lrtrim(str, trimStr) {
+    if (typeof str === "string") {
```

Comment:
Although it's not painfully obvious here because we only have a single `if` but we should avoid nesting the logic in such a condition because the nesting can get deep and messy if more conditions are required. We should instead account for all the edge cases first and return early. So in this case we could just have

```
if (typeof str !== "string") {
  return str;
}
```